### PR TITLE
Feat/ga/unit testing setup 110

### PR DESCRIPTION
**Pull request by unseen1980**
https://github.com/SAP/guided-answers-extension/issues/110

Setup and configure jest and enzyme libraries in webapp so that we start adding unit tests for React components
Screenshot 2022-06-09 at 16 42 15

### CODE REVIEW

File: packages/webapp/package.json [hunk 1]
```diff
@@ -31,6 +34,8 @@
         "esbuild-plugin-svgr": "1.0.1",
         "esbuild-sass-plugin": "2.2.4",
         "html-react-parser": "1.4.8",
+        "jest-css-modules-transform": "4.4.2",
+        "jest-sonar-reporter": "2.0.0",
```
Comment (Klaus-Keller) [inline packages/webapp/package.json:38]: Hi @unseen1980, we have `jest-sonar` module as `devDependency` in root `package.json`. In PR https://github.com/SAP/guided-answers-extension/pull/113 I'm currently working on enabling SonarCloud scans and modify the `jest.config.js` files to use `jest-sonar`.
Do we also need `jest-sonar-reporter` ?

**Review by Klaus-Keller — COMMENTED** [review]
@unseen1980, test setup looks good and works for me. Small question regarding `jest-sonar-reporter`.

File: packages/webapp/package.json (hunk 1 above)
Comment (unseen1980) [reply packages/webapp/package.json:38]: No we don't, I added it because of 
```testResultsProcessor: 'jest-sonar-reporter',```. I'll remove it.

**Review by Klaus-Keller — APPROVED** [review]

**Comment by unseen1980** [comment]
Thanks @Klaus-Keller

**Pull request merged**